src/main/resources/db/migration/V1__Init.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE letter (
+  id UUID NOT NULL PRIMARY KEY,
+  service VARCHAR(256) NOT NULL,
+  created TIMESTAMP NOT NULL,
+  sent_to_print TIMESTAMP NULL,
+  printed TIMESTAMP NULL
+);
+
+CREATE TABLE letter_metadata (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  letter_id UUID NOT NULL,
+  metadata VARCHAR(256) NOT NULL,
+  metadata_value VARCHAR(256) NOT NULL
+);
```

Comment:
we can simply store additional data in json column

Reply:
how will querying work? is json type accommodated to perform efficiently?

Reply:
There are the `->` and `->>` operators

Reply:
the future 🐙